### ISSUE TITLE
Adress type hint errors in util/node.py

### ISSUE THREAD

**Issue by LauWien** [iiasa/message-ix-models#38]
Possible due to the new [release](https://pypi.org/project/mypy/) of mypy on Dec 16th the following errors occured in the lint workflow of `message_data`:
```
message-ix-models/message_ix_models/util/node.py:46: error: Argument 1 to "adapt_R11_R14" has incompatible type "Union[Any, Quantity]"; expected "Dict[str, Union[Any, Quantity]]"
message-ix-models/message_ix_models/util/node.py:85: error: Dispatch type "Quantity" must be subtype of fallback function first argument "Dict[str, Union[Any, Quantity]]"
```
To adress these errors the two mentioned lines of the module node.py need to be adapted.

**Comment by LauWien** [comment]
As discussed, and I hope I remember correctly, I assigned you @khaeru, but also happy to look into it if something changes.

**Comment by khaeru** [comment]
This was closed via #39.
